.github/workflows/c8run-release.yaml

```diff
@@ -168,8 +171,34 @@ jobs:
             ${{ env.CAMUNDA_RUN_NAME_ARTIFACT_WITH_PATCH_VERSION }}*
         env:
           GH_TOKEN: ${{ github.token }}
-      - name: Add release URL to workflow summary
+      - name: Upload artifact to Camunda download center
+        uses: camunda/infra-global-github-actions/download-center-upload@main
+        with:
+          gcp_credentials: ${{ steps.secrets.outputs.GCP_CREDENTIALS_NAME }}
+          env: 'prod'
+          version: ${{ inputs.camundaAppsRelease }}
+          artifact_file: ./c8run/camunda8-run-*
```

Comment:
I checked with https://downloads.camunda.cloud/release/camunda/c8run/8.7.0-alpha5/ and noticed that the files are all quite large with over ~1GB each, is that intended?

Reply:
fyi @camunda/infra-finops-dri (@clementnero) - in case we start seeing more traffic-related costs for Download Center, it may be related to the addition of C8Run to it (expected)